static/js/sefaria/hebrew.js

```diff
@@ -20,19 +21,109 @@ class Hebrew {
   /**
    * Encodes an integer "Daf" and returns a string encoding it as a Hebrew numeral.
    * @param {number} n - The integer
-   * @returns {string} - Hebrew numeral
+   * @returns {number[]} - Hebrew numeral
    */
-  static encodeHebrewNumeral(n) {
+  static breakIntMagnitudes(n, start=null) {
+    /* Accepts an integer and an optional integer (multiple of 10) for at what order of
+	magnitude to start breaking apart the integer.  If no option "start" is provided,
+	function will determine the size of the input integer and start that the largest order
+	of magnitude.
+	Returns a big-endian list of the various orders of magnitude, by 10s, broken apart.
+
+	breakIntMagnitudes(1129, 100)
+	[1100, 20, 9]
+
+	breakIntMagnitudes(2130)
+	[2000, 100, 30, 0]
+
+	breakIntMagnitudes(15000)
+	[10000, 5000, 0, 0, 0]
+     */
+    if (!!start) {
+      if (!(start % 10 === 0 || start === 1)) {
+        throw new TypeError(`Argument 'start' must be 1 or divisible by 10, ${start} provided.`);
+      }
+    } else {
+      start = Math.pow(10, Math.floor(Math.log10(n)));
```

Comment:
it seems like start can have a default of 10 instead of null and that way this line can just be taken care of in the `const thisIntMagnitude = Math.floor(n / start) * start;`
Am I wrong?

Reply:
I'm not sure what you mean.  If 'n' is 1250, start should be 1000, and if 'n' is 120, 'start' should be 100.  Why should it default to 10?